auth-web/src/components/auth/TransactionsDataTable.vue

```diff
@@ -0,0 +1,218 @@
+      <v-list-item-subtitle
+        :data-test="getIndexedTag('transaction-sub', item.index)"
+      >Business Registry</v-list-item-subtitle>
+    </template>
+    <template v-slot:item.transactionDate="{ item }">
+      {{formatDate(item.transactionDate)}}
+    </template>
+    <template v-slot:item.totalAmount="{ item }">
+      <div class="font-weight-bold">
+        ${{item.totalAmount.toFixed(2)}}
+      </div>
+    </template>
+    <template v-slot:item.status="{ item }">
+      <div
+        class="font-weight-bold"
+        v-bind:class="getStatusClass(item)"
+      >
+        {{item.status}}
+      </div>
+    </template>
+  </v-data-table>
+</template>
+
+<script lang="ts">
+import { Account, TransactionStatus } from '@/util/constants'
+import { Component, Emit, Prop, Vue } from 'vue-property-decorator'
+import { Member, MembershipStatus, MembershipType, Organization, RoleInfo } from '@/models/Organization'
+import { Transaction, TransactionDateFilter, TransactionListResponse, TransactionTableList, TransactionTableRow } from '@/models/transaction'
+import { mapActions, mapState } from 'vuex'
+import { Business } from '@/models/business'
+import CommonUtils from '@/util/common-util'
+
+@Component({
+  computed: {
+    ...mapState('org', [
+      'currentOrganization'
+    ])
+  },
+  methods: {
+    ...mapActions('org', [
+      'getTransactionList'
+    ])
+  }
+})
+export default class TransactionsDataTable extends Vue {
+  private readonly currentOrganization!: Organization
+  private readonly getTransactionList!: (dateFilter: TransactionDateFilter) => TransactionTableList
+
+  private transactionList: TransactionTableRow[] = [];
+  private formatDate = CommonUtils.formatDisplayDate
+  private readonly ITEMS_PER_PAGE = 10
+
+  private readonly headerTranscations = [
+    {
+      text: 'Transaction',
+      align: 'left',
+      sortable: true,
+      value: 'transactionNames'
+    },
+    {
+      text: 'Folio #',
+      align: 'left',
+      sortable: true,
+      value: 'folioNumber'
+    },
+    {
+      text: 'Initiated By',
+      align: 'left',
+      sortable: true,
+      value: 'initiatedBy'
+    },
+    {
+      text: 'Date',
+      align: 'left',
+      value: 'transactionDate',
+      sortable: true
+    },
+    {
+      text: 'Total Amount',
+      align: 'left',
+      value: 'totalAmount',
+      sortable: true
+    },
+    {
+      text: 'Status',
+      align: 'left',
+      value: 'status',
+      sortable: true
+    }
+  ]
+
+  private readonly transactionStatus = [
+    {
+      status: TransactionStatus.COMPLETED,
+      description: 'Funds received'
+    },
+    {
+      status: TransactionStatus.CREATED,
+      description: 'Transaction Created'
+    },
+    {
+      status: TransactionStatus.DELETED,
+      description: 'Transaction Deleted'
+    }
+  ]
+
+  private get getPaginationOptions () {
+    let pagination = [...Array(4)].map((value, index) => this.ITEMS_PER_PAGE * (index + 1))
```

Comment:
more comments plz .. like why 4.. or name 4 as MAXIMUM_PAGE_SIZE or something... wil be easily readable..
like ITEMS_PER_PAGE ..